drivers/timer/xtensa_sys_timer.c

```diff
@@ -433,11 +433,16 @@ void _zxt_tick_timer_init(void)
 	int val;
 
 	__asm__ volatile("rsr.intenable %0" : "=r"(val));
+
+	/*
+	 * Since CCOMPARE* registers have undefined values after reset,
+	 * set compare value first beforing enabling timer interrupt.
+	 */
+	SET_TIMER_FIRE_TIME(GET_TIMER_CURRENT_TIME() + _xt_tick_divisor);
```

Comment:
Can you yank this one out of the PR?  I'm finishing up a completely new xtensa timer driver and am too lazy to rebase. 

I promise it will initialize the CCOMPARE register correction. :)

Reply:
Done.